src/components/Header.jsx

```diff
@@ -8,6 +8,10 @@ import { connect } from 'react-redux'
 import _ from 'lodash'
 
 class Header extends Component {
+  hidePopovers() {
+    window.$('.popover').popover('hide')
```

Comment:
this works fine in browser, but makes the tests unhappy :(

Reply:
Maybe this?
```
if(window.$('.popover').popover) {
      window.$('.popover').popover('hide')
    }
```

Reply:
that worked, thanks